docs/push_by_github_action-zh.md

```diff
@@ -0,0 +1,122 @@
+# 如何在 github action 中使用 kpm 发布您的 KCL 包 
+
+[kpm](https://github.com/KusionStack/kpm) 是一个用于管理 kcl 包的工具。本文将指导您如何在 GitHub Action 中使用 kpm 将您的 kcl 包推送到发布到 ghcr.io 中。
+
+下面是一个简单的步骤，指导您如何使用 kpm 将您的 kcl 包推送到 OCI Registry。
+
+## 步骤 1：安装 kpm
+
+首先，您需要在您的计算机上安装 kpm。您可以按照 [kpm 安装文档](https://kcl-lang.io/docs/user_docs/guides/package-management/installation)中的说明进行操作。
+
+## 步骤 2：创建一个 GitHub 账号
+
+如果您已经有 GitHub 帐号了，您可以选择跳过这一步
+
+[注册新的一个 GitHub 账号](https://docs.github.com/zh/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+
+## 步骤 3: 为您的 KCL 包创建一个 GitHub 仓库并进行相关配置
+
+您需要为您的 KCL 程序包准备一个 GitHub 仓库。
+
+[创建一个GitHub仓库](https://docs.github.com/zh/get-started/quickstart/create-a-repo) 
+
+您需要为您的 GitHub Action 准备一个 GitHub Token，用于向 ghcr.io 写入内容。
+
+[创建一个 GitHub Token](https://docs.github.com/zh/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#personal-access-tokens-classic)
+
+在仓库中添加 token 作为 secrets。
+
+[为仓库添加 secrets](https://docs.github.com/zh/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository)
+
+将我们添加的 token 命令为 `REG_TOKEN`.
+
+## 步骤 4: 将您的 KCL 包添加到仓库中并编写 github action workflow
```

Comment:
How to add the KCL package into this repo and the what is the file structure? It is better to give a example.